packages/dumb/src/utils/datetime/datetime.util.ts

```diff
@@ -129,66 +79,20 @@ export const getBrowserTimezone = (): string => {
  *
  * @return {String} timezoneName from timePrefs, browser, or fallback to 'UTC'
  */
-export const getTimezoneFromTimePrefs = (timePrefs: { timezoneAware: boolean, timezoneName: string }): string => {
+export const getTimezoneFromTimePrefs = (timePrefs?: { timezoneAware?: boolean, timezoneName?: string }): string => {
```

Comment:
Would it be very complicated to reuse `TimePrefs` type here? You can even use `Partial<TimePrefs>` if you're not sure that any of the fields will be there